<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMyTableWidget</class>
 <widget class="QMainWindow" name="QMyTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QMyTableWidget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1470</width>
      <height>914</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>650</width>
             <height>900</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="title">
            <string>表格格式设置</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="FormEditedEnabled">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>表格可编辑</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="ShowLineHeader">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>显示行表头</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="RowSelection">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>行选择</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="IntervalRowBackgroundColor">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>间隔行底色</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="ShowListHeader">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>显示列表头</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="CellSelection">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>单元格选择</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <layout class="QFormLayout" name="formLayout_2">
              <property name="horizontalSpacing">
               <number>10</number>
              </property>
              <property name="verticalSpacing">
               <number>10</number>
              </property>
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout_3">
                <property name="horizontalSpacing">
                 <number>6</number>
                </property>
                <property name="verticalSpacing">
                 <number>10</number>
                </property>
                <item row="1" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="spacing">
                   <number>10</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <property name="spacing">
                     <number>10</number>
                    </property>
                    <item>
                     <widget class="QPushButton" name="SetRowsNum">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>80</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Times New Roman</family>
                        <pointsize>16</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>设置行数</string>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                      <property name="default">
                       <bool>false</bool>
                      </property>
                      <property name="flat">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="DeleteCurrentRow">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>80</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Times New Roman</family>
                        <pointsize>16</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>删除当前行</string>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                      <property name="default">
                       <bool>false</bool>
                      </property>
                      <property name="flat">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="InsertRow">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>80</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Times New Roman</family>
                        <pointsize>16</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>插入行</string>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                      <property name="default">
                       <bool>false</bool>
                      </property>
                      <property name="flat">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="AutoAdjustRowHeight">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>80</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Times New Roman</family>
                        <pointsize>16</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>自动调节行高</string>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                      <property name="default">
                       <bool>false</bool>
                      </property>
                      <property name="flat">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <property name="spacing">
                     <number>10</number>
                    </property>
                    <item>
                     <widget class="QSpinBox" name="spinBox">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>80</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Times New Roman</family>
                        <pointsize>16</pointsize>
                       </font>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="AddRow">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>80</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Times New Roman</family>
                        <pointsize>16</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>添加行</string>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                      <property name="default">
                       <bool>false</bool>
                      </property>
                      <property name="flat">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="ClearTabContent">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>80</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Times New Roman</family>
                        <pointsize>16</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>清空表格内容</string>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                      <property name="default">
                       <bool>false</bool>
                      </property>
                      <property name="flat">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="AutoAdjustColWidth">
                      <property name="minimumSize">
                       <size>
                        <width>240</width>
                        <height>80</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Times New Roman</family>
                        <pointsize>16</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>自动调节列宽</string>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                      <property name="default">
                       <bool>false</bool>
                      </property>
                      <property name="flat">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="SetHeader">
                  <property name="minimumSize">
                   <size>
                    <width>616</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>设置表头</string>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                  <property name="default">
                   <bool>false</bool>
                  </property>
                  <property name="flat">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="ReadTabContentToText">
                <property name="minimumSize">
                 <size>
                  <width>616</width>
                  <height>80</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>读取表格内容到文本</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="InitialForm">
              <property name="minimumSize">
               <size>
                <width>616</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>初始化表格</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>450</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="TabWidget">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>450</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>13</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
